libtransmission/utils.cc

```diff
@@ -257,8 +257,8 @@ uint64_t tr_time_msec()
  */
 size_t tr_strlcpy(void* vdst, void const* vsrc, size_t siz)
 {
-    auto* dst = static_cast<char*>(vdst);
-    auto const* const src = static_cast<char const*>(vsrc);
+    auto* dst = reinterpret_cast<char*>(vdst);
+    auto const* const src = reinterpret_cast<char const*>(vsrc);
```

Comment:
I don't think this is correct; `static_cast` is a smaller cast and is fine for the special case of void pointers.

Reply:
Huh that's apparently true. Then my other PR should be declined without a question.